pages/reference/OS/updates/self-service.md

```diff
@@ -7,14 +7,18 @@ excerpt: How to update {{ $names.os.lower }} versions from your dashboard
 
 ## Which devices and versions are supported?
 
-Since we periodically release updates and improvements to {{ $names.os.lower }} (the host OS running on all {{ $names.company.lower }} devices), we encourage you to keep your devices up to date. We offer self-service host OS updates between 2.x versions and from 1.x to 2.x versions. All 2.x devices will default to delta-based updates if available, thus reducing the size of the update sent over the network.
+We periodically release updates and improvements to balenaOS, the host operating system that powers all balena devices. These releases are accessible through multiple channels, including the dashboard, SDK, and CLI.
 
-__Note:__ {{ $names.os.upper }} 1.x to 2.x updates limit the amount of data you can have in your application's `/data` folder to about 170MB (compressed). If you have more data, the update will fail and your device won't be modified.
+Updating from one version to a newer one is generally supported; however, big version jumps may not always be guaranteed. Devices running balenaOS 2.x and above benefit from delta-based updates when available, significantly reducing the size of updates sent over the network.
 
 Self-service updates are available for both `production` and `development` {{ $names.os.lower }} variants.
 
 For device types and {{ $names.os.lower }} versions that are not yet supported, please contact us on the [forums][forums]. We are continuously expanding the range of versions and types that can be updated.
 
+Find more information below about our version support policy.
+
+__Note:__ {{ $names.os.upper }} 1.x to 2.x updates limit the amount of data you can have in your application's `/data` folder to about 170MB (compressed). If you have more data, the update will fail and your device won't be modified.
+
```

Comment:
This note is probably no longer relevant as we should have no devices lower than v2.14 on the fleets.